content/working_with/console/widgets/blueprints.md

```diff
@@ -53,6 +53,50 @@ There are also action buttons to upload a blueprint, create deployment, delete b
 The default visibility is "Tenant", and according to the logged-in user's permissions you can also choose other levels of [resource visibilities]({{< relref "working_with/manager/resource-visibility.md" >}}).<br>
 6. Click **Upload**.
 
+##### Using Terraform module
+1. Click the **Upload** button.
+2. In the menu, click **Upload from Terraform module** option.
+3. **Create blueprint from Terraform** modal will appear.
+4. Enter the `Blueprint name`   
+   `Blueprint name` is the name with which you want to identify this blueprint once uploaded.
+5. Select `Terraform version` (by default, one of the Terraform versions will be selected).
+6. Provide `Blueprint main information`, which consists of:
+   - `URL to a zip archive that contains the Terraform module`.
+   - `Terraform folder in the archive` - it's a selectable field from which you'll be able to select directories found inside the linked zip file.<br />
+   By default this field is disabled, until the `URL to a zip archive that contains the Terraform module` is provided.
+   - (Optional) Credentials required for accessing `URL to a zip archive that contains the Terraform module`.
+7. (Optional) To add `Variables`:   
+   1. Click the `Variables` select.
+   2. Click the **Add** button.
+   3. Filling the row fields:
+      - `Variable` - name of the variable.
+      - `Source` - type of the variable, which may be selected from a dropdown.<br />
+         Currently available values are: `Secret`, `Input`, `Static`.
+      - `Value / Secret key / Input name` - value of the variable (related to the selected `Source`).<br />
+         Selecting `Static` as a `Source` enables to type a value. <br />
+         Selecting `Input` as a `Source` enables to type a value. <br />
+         Selecting `Secret` enables to select a value from one of stored secrets.
+8. (Optional) To add `Environment variables`:   
+   1. Click the `Environment variables` select.
```

Comment:
I see that my comment related to merging "Environment variables" and "Variables" has gone.

I think it would still be better to have it merged like that - instead of having:
```suggestion
8. (Optional) To add `Variables` or `Environment variables`:   
   1. Click the `Variables or `Environment variables` section.
```

That way you would be able to reduce ~10 lines.

Reply:
Done 🎉